source/conf.py

```diff
@@ -82,3389 +82,1708 @@ def setup(_: Sphinx):
-        "https://docs.mattermost.com/collaborate/keyboard-shortcuts.html#navigation-in-the-desktop-app",
-"welcome/keyboard-shortcuts.html#zoom-shortcuts-for-web-and-desktop-app":
-        "https://docs.mattermost.com/collaborate/keyboard-shortcuts.html#zoom-in-zoom-out-display",
-"welcome/log-out.html#frequently-asked-questions":
-        "https://docs.mattermost.com/collaborate/log-out.html#frequently-asked-questions",
-"welcome/log-in.html#reset-your-password":
-        "https://docs.mattermost.com/collaborate/access-your-workspace.html#reset-your-password",
-"welcome/log-in.html#email-address-or-username":
-        "https://docs.mattermost.com/collaborate/access-your-workspace.html#email-address-or-username",
-"welcome/log-in.html#single-sign-on-sso":
-        "https://docs.mattermost.com/collaborate/access-your-workspace.html#single-sign-on-sso",
-"welcome/get-started-mattermost-channels.html#channels-and-direct-messages":
-        "https://docs.mattermost.com/collaborate/collaborate-within-channels.html#you-re-added-to-default-channels-automatically",
-"welcome/get-started-mattermost-channels.html#create-and-join-channels":
-        "https://docs.mattermost.com/collaborate/collaborate-within-channels.html#learn-more",
-"welcome/get-started-mattermost-channels.html#invite-people-to-the-team":
-        "https://docs.mattermost.com/collaborate/invite-people.html",
-"welcome/get-started-mattermost-channels.html#send-messages":
-        "https://docs.mattermost.com/collaborate/send-messages.html",
-"welcome/get-started-mattermost-channels.html#customize-your-mattermost-experience":
-        "https://docs.mattermost.com/guides/preferences.html",
-"welcome/manage-custom-groups.html#create-a-custom-group":
-        "https://docs.mattermost.com/collaborate/organize-using-custom-user-groups.html#create-a-custom-group",
-"welcome/manage-custom-groups.html#review-group-members":
-        "https://docs.mattermost.com/collaborate/organize-using-custom-user-groups.html#review-group-members",
-"welcome/manage-custom-groups.html#manage-custom-user-groups":
-        "https://docs.mattermost.com/collaborate/organize-using-custom-user-groups.html#manage-custom-user-groups",
-"welcome/add-people.html#add-people-on-demand":
-        "https://docs.mattermost.com/collaborate/invite-people.html",
-"welcome/sign-in.html":
-        "https://docs.mattermost.com/collaborate/access-your-workspace.html",
-"welcome/sign-in.html#email-address-or-username":
-        "https://docs.mattermost.com/collaborate/access-your-workspace.html#email-address-or-username",
-"welcome/sign-in.html#reset-your-password":
-        "https://docs.mattermost.com/collaborate/access-your-workspace.html#reset-your-password",
-"welcome/sign-in.html#single-sign-on-sso":
-        "https://docs.mattermost.com/collaborate/access-your-workspace.html#single-sign-on-sso"
-
-# End of redirects. The last redirect above should NOT end in a comma.
-
+    # How to add redirects for docs.mattermost.com:
```

Comment:
Not sure what's going on in this file

Reply:
This is where we define and manage page redirects to ensure that docs visitors reach the correct page in cases where pages move, are archived, or are otherwise no longer available at the original URL.